registry.json

```diff
@@ -2954,5 +2954,14 @@
     "telegram": "StakeWithPride",
     "operatorName": "",
     "github": ""
-  }
+  },
+  {
+ "ticker": "SOLRP",
+ "poolId": "9728b10a926c048af938e5c52053319db5be921e8b698842c3afd3cc",
+ "social": {
+   "twitter": "SOLRP_StakePool"
```

Comment:
Please can you add the other social keys, e.g. reddit, website etc? They need to be present, just leave them blank if you do not have those socials.

Reply:
Added.